catalog-management/v1.ts

```diff
@@ -4192,6 +4213,67 @@ class CatalogManagementV1 extends BaseService {
     return this.createRequest(parameters);
   }
 
+  /**
+   * Validates deployment input variables.
+   *
+   * Validates deployment input variables.
+   *
+   * @param {Object} params - The parameters to send to the service.
+   * @param {string} params.versionLocId - A dotted value of `catalogID`.`versionID`.
+   * @param {string} [params.input1] - A deployment variable to validate.
+   * @param {string} [params.input2] - Another deployment variable to validate.
+   * @param {OutgoingHttpHeaders} [params.headers] - Custom request headers
+   * @returns {Promise<CatalogManagementV1.Response<CatalogManagementV1.VersionInputValidationResponse>>}
+   */
+  public validatesInputs(
```

Comment:
This seems like an odd name for an operation.  Perhaps `validateInputs` would be a slightly better name?